modules/readJSON.js

```diff
@@ -14,5 +21,14 @@ exports.readJSON = function() {
 	})
 	pics = data.images
 	removing = data.filterlist
+	// read in the key-value pairs
+	timezonesJSON = data.timezones
+	timezones = []
+	// put each pair into a class
+	for (const [key, value] of Object.entries(timezonesJSON)) {
+  	let i = new userTz(key, value);
+		// and push it to the new array
+		timezones.push(i)
+	}
```

Comment:
looping back to the unneeded class, this would get changed without that class

Reply:
see earlier